larq/metrics.py

```diff
@@ -87,18 +135,24 @@ def __init__(
         super().__init__(name=name, dtype=dtype)
         self.values_dtype = tf.as_dtype(values_dtype)
         self.values_shape = tf.TensorShape(values_shape).as_list()
+        self.is_weight_metric = True
```

Comment:
This isn't used anywhere. Or am I missing something?
```suggestion
```

Reply:
Not in this PR, but I need it for the MetricLayer. Didn't want to push them all at the same time since the changes might be a bit overwhelming then